public/lib/events.js

```diff
@@ -34,7 +34,7 @@ var events = {
 
       // if this was a published event, do an immediate trigger
       var cache = self._cache;
-      if (cache[name]) {
+      if (cache && cache[name]) {
```

Comment:
but cache is created in the constructor.. even though this might be a fix, it's definitely not here where the problem is.